kolibri/utils/options.py

```diff
@@ -15,6 +15,7 @@
 from configobj import ConfigObj
 from configobj import flatten_errors
 from configobj import get_extra_values
+from django.core.files.storage import Storage
```

Comment:
This is probably why flake8 wouldn't let pre-commit pass but it didn't give me useful output